decidim-core/app/helpers/decidim/paginate_helper.rb

```diff
@@ -13,7 +13,7 @@ def decidim_paginate(collection, paginate_params = {})
 
       content_tag :div, class: "flex flex-col-reverse md:flex-row items-center justify-between gap-1 py-8 md:py-16", data: { pagination: "" } do
         template = ""
-        template += render partial: "decidim/shared/results_per_page", formats: ["html"] if collection.total_pages.positive?
+        template += render(partial: "decidim/shared/results_per_page", formats: [:html]) if collection.total_pages.positive?
```

Comment:
Related to #12035, seems like in Rails 6 we got away with using a string, but now we need to use a symbol, or it failed in certain scenarios where this was called.